src/release/breaking-changes/index.md

```diff
@@ -37,13 +37,15 @@ release, and listed in alphabetical order:
 * [Deprecate `describeEnum` and update `EnumProperty` to be type strict][]
 * [Deprecated just-in-time navigation pop APIs for Android Predictive Back][]
 * [Updated default text styles for menus][]
+* [Introduce FlutterEngine::ProcessExternalWindowMessage][]
```

Comment:
Ideally this line would make it a little clearer who is affected by this break: Windows app that have non-Flutter windows. What do you think of a title like `[Windows] External windows should notify Flutter of top-level window messages`? I'm not 100% happy with that either, so feel free to use something else :)